srl/pom.xml

```diff
@@ -0,0 +1,165 @@
+        <artifactId>illinois-cogcomp-nlp</artifactId>
+        <groupId>edu.illinois.cs.cogcomp</groupId>
+        <version>3.0.73</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>illinois-srl</artifactId>
+    <packaging>jar</packaging>
+    <url>http://cogcomp.cs.illinois.edu</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <cogcomp-nlp-version>3.0.59</cogcomp-nlp-version>
+        <cogcomp-nlp-pipeline-version>0.1.24</cogcomp-nlp-pipeline-version>
+    </properties>
+
+    <dependencies>
+        <!-- Include the pre-trained SRL models for running SemanticRoleLabeler -->
+        <!-- Notice that the models need to match up to the minor version number -->
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-srl-models</artifactId>
+            <classifier>verb-stanford</classifier>
+            <version>5.1</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-srl-models</artifactId>
+            <classifier>nom-stanford</classifier>
+            <version>5.1</version>
+        </dependency>
+
+        <!--The Illinois pipeline can be used instead -->
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-nlp-pipeline</artifactId>
+            <version>${cogcomp-nlp-pipeline-version}</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>illinois-srl</artifactId>
+                    <groupId>edu.illinois.cs.cogcomp</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <!-- The following 3 projects are now developed under illinois-cogcomp-nlp -->
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-core-utilities</artifactId>
+            <version>${cogcomp-nlp-version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-curator</artifactId>
+            <version>${cogcomp-nlp-version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-edison</artifactId>
+            <version>${cogcomp-nlp-version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.gurobi</groupId>
+            <artifactId>gurobi</artifactId>
+            <version>6.5</version>
+            <optional>true</optional>
+        </dependency>
+
+        <dependency>
+            <groupId>edu.illinois.cs.cogcomp</groupId>
+            <artifactId>illinois-common-resources</artifactId>
+            <classifier>illinoisSRL</classifier>
```

Comment:
<img width="531" alt="screen shot 2016-09-29 at 1 29 46 pm" src="https://cloud.githubusercontent.com/assets/2441454/18966825/5d9fcf36-8648-11e6-8084-36ff728efd74.png">

[1.3 is the one currently used in Edison](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/blob/master/edison/pom.xml#L20-L25) and [1.4 is the one used in SRL](https://gitlab-beta.engr.illinois.edu/cogcomp/illinois-srl/blob/master/pom.xml#L82-87). There is an effort to break the resources into smaller jar files. [I can update it after that.](https://github.com/IllinoisCogComp/illinois-cogcomp-nlp/issues/186)